components/ui/themes/PurpleTheme.ts

```diff
@@ -35,9 +35,10 @@ const Colors = {
   DisabledColors: ['#C7C7C7', '#C7C7C7'],
   captureIconBorder: '#F59B4B',
   Purple: '#70308C',
-  LightPurple: '#AEA7FF',
-  TimeoutHintBoxColor: '#FFF7E5',
-  TimoutHintText: '#8B6105',
+  LightPurple: '#F3E2FF',
+  TimeoutHintBoxColor: '#FBF5FF',
+  TimeoutHintBoxBorder: '#FFF7E5',
```

Comment:
TimeoutHintBoxBorder property is not added in default Theme.ts. 

please try to have the property added in both the themes, so that on switching the theme it will be consistant.

Reply:
Done